apps/02RockPaperScissors.js

```diff
@@ -7,8 +7,34 @@ prompt.start();
 
 function rockPaperScissors(hand1, hand2) {
 
-    // Write code here
+  hand1 = hand1.toLowerCase();
+  hand2 = hand2.toLowerCase();
 
+  if (hand1 === hand2) {
+    return "It's a tie!";
+  };
+
+  if ((hand1 === "rock") && (hand2 === "scissors")) {
```

Comment:
Nice! This could be a little shorter, but it's nice and clear how it's laid out. Readability is key!